src/NLog/Targets/Wrappers/AsyncTargetWrapper.cs

```diff
@@ -222,7 +218,32 @@ protected virtual void StartLazyWriterTimer()
             {
                 if (this.lazyWriterTimer != null)
                 {
-                    this.lazyWriterTimer.Change(this.TimeToSleepBetweenBatches, Timeout.Infinite);
+                    if (this.TimeToSleepBetweenBatches <= 0)
+                    {
```

Comment:
Added Trace-Message when this.TimeToSleepBetweenBatches <= 0. The other code-path is known and tested.